Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build Geth in a stock Go builder container
-FROM golang:1.12-alpine as builder
+FROM golang:1.12.1-alpine as builder
```

Comment:
1.12 is a wildcard for the latest version in the 1.12.x family. You don't need to explicitly specify the patch release version. Please revert this change.

```
$ docker pull golang:1.12-alpine
1.12-alpine: Pulling from library/golang
8e402f1a9c57: Pull complete 
ce7779d8bfe3: Pull complete 
de1a1e452942: Pull complete 
a7ffcdf77d94: Pull complete 
e7348c6471e5: Pull complete 
Digest: sha256:5f7781ceb97dd23c28f603c389d71a0ce98f9f6c78aa8cbd12b6ca836bfc6c6c
Status: Downloaded newer image for golang:1.12-alpine

$ docker run golang:1.12-alpine go version
go version go1.12.1 linux/amd64
```